src/test/resources/graphql/attributions.txt

```diff
@@ -0,0 +1,18 @@
+query ($namespace: String) {
```

Comment:
Seeing this GraphQL reminded me that we should probably add snapshot tests to the GraphQL schema in gtfs-lib for these things.